homeassistant/components/onkyo/config_flow.py

```diff
@@ -46,13 +47,11 @@
 INPUT_SOURCES_ALL_MEANINGS = [
     input_source.value_meaning for input_source in InputSource
 ]
+STEP_MANUAL_SCHEMA = vol.Schema({vol.Required(CONF_HOST): str})
 STEP_CONFIGURE_SCHEMA = vol.Schema(
     {
-        vol.Required(
-            OPTION_VOLUME_RESOLUTION,
-            default=OPTION_VOLUME_RESOLUTION_DEFAULT,
-        ): vol.In(VOLUME_RESOLUTION_ALLOWED),
-        vol.Required(OPTION_INPUT_SOURCES, default=[]): SelectSelector(
+        vol.Required(OPTION_VOLUME_RESOLUTION): vol.In(VOLUME_RESOLUTION_ALLOWED),
+        vol.Required(OPTION_INPUT_SOURCES): SelectSelector(
```

Comment:
Why do we have an option for input sources (besides there was this option in YAML)?

Reply:
Doesn't the device tell us what are present?

Reply:
No, it does not.

There is a discovery mechanism for that, but it is not implemented by older receivers, and even for newer ones it is still not supported by the library we are using.

Reply:
Ok.